util/templates/types.go

```diff
@@ -183,17 +183,18 @@ type LinkedTemplate struct {
 // 3. defaults.yaml modbus section
 // 4. template
 type Param struct {
```

Comment:
It would be nice if we didn't change the entire struct- we have open PRs looking at the same spot :O